trade-finance-manager-ui/templates/case/documents/documents.njk

```diff
@@ -2,6 +2,8 @@
 
 {% extends "case/case.njk" %}
 
+{% block pageTitle -%}Deal {{ deal.details.ukefDealId }} - Documents{%- endblock %}
```

Comment:
Multiline formatting please.

Reply:
Changed all pageTitle blocks to multiline format.